src/Configuration.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace Wheniwork\Feedback;
+
+use Auryn\Injector;
+
+class Configuration
+{
+    public function apply(Injector $injector, array $env)
+    {
+        $injector->define("Predis\Client", []);
+        $injector->define("GuzzleHttp\Client", []);
+
+        // --------------------
+        // Initialize services
+        // --------------------
+        $services = "Wheniwork\Feedback\Service";
+
+        $injector->define("$services\Authorizer", [
+            ':key' => $env['POST_KEY']
+        ]);
+
+        $injector->define("$services\AppStoreService", [
+            ':app_id' => $env['WIW_IOS_APP_ID']
+        ]);
+
+        $injector->define("\HieuLe\WordpressXmlrpcClient\WordpressClient", [
+            ':xmlrpcEndPoint' => "http://wheniwork.com/blog/xmlrpc.php",
+            ':username' => $env['WP_USER'],
+            ':password' => $env['WP_PASSWORD']
+        ]);
+        $injector->define("$services\BlogService", [
+            ':users' => preg_split('/\s*,\s*/', $env['WP_FEEDBACK_USERS'])
+        ]);
+
+        $injector->define("Facebook\Facebook", [
+            ':config' => [
+                'app_id' => $env['FB_APP_ID'],
+                'app_secret' => $env['FB_APP_SECRET'],
+                'default_graph_version' => 'v2.4'
+            ]
+        ]);
+        $injector->define("$services\FacebookService", [
+            ':page_id' => $env['FB_PAGE_ID']
+        ]);
+        $injector->prepare("$services\FacebookService", function($service) use ($env) {
+            $service->authenticate(
+                $env['FB_APP_ID'],
+                $env['FB_APP_SECRET']
+            );
+        });
+
+        $injector->define("\Github\Client", []);
+        $injector->prepare("\Github\Client", function($client) use ($env) {
+            $client->authenticate($env['GITHUB_TOKEN'], "", \Github\Client::AUTH_HTTP_TOKEN);
```

Comment:
I think you could use `$client::AUTH_HTTP_TOKEN` here instead to avoid referencing the class twice.